synapse/cortex.py

```diff
@@ -156,6 +157,31 @@
     ],
 })
 
+reqValidVault = s_config.getJsValidator({
+    'type': 'object',
+    'properties': {
+        'name': {'type': 'string', 'pattern': '^.{1,128}$'},
+        'iden': {'type': 'string', 'pattern': s_config.re_iden},
+        'type': {'type': 'string', 'pattern': '^.{1,128}$'},
+        'scope': {'type': ['string', 'null'], 'enum': [None, 'user', 'role', 'global']},
+        'owner': {'type': ['string', 'null'], 'pattern': s_config.re_iden},
+        'permissions': s_msgpack.deepcopy(s_cell.easyPermSchema),
```

Comment:
is there a reason we cannot just us `copy.deepcopy()` here?

Reply:
@invisig0th said the msgpack version is faster so I used that.